packages/lab/src/typography/Text.css

```diff
@@ -144,3 +144,7 @@ label.uitkText,
   line-height: var(--uitk-text-figure-3-lineHeight);
   font-weight: var(--uitk-typography-weight-semiBold);
 }
+
+.backwardsCompat {
```

Comment:
Should this have any other class? otherwise this will impact all backwards compat?